src/DataWrangling/ECCO/ECCO_metadata.jl

```diff
@@ -121,10 +113,17 @@ ECCO_location = Dict(
     :v_velocity            => (Center, Face,   Center),
 )
 
+const ECCO2_url = "https://ecco.jpl.nasa.gov/drive/files/ECCO2/cube92_latlon_quart_90S90N/"
+
 # URLs for the ECCO datasets specific to each dataset
-urls(::Metadata{<:Any, <:ECCO2Monthly}) = "https://ecco.jpl.nasa.gov/drive/files/ECCO2/cube92_latlon_quart_90S90N/monthly"
-urls(::Metadata{<:Any, <:ECCO2Daily})   = "https://ecco.jpl.nasa.gov/drive/files/ECCO2/cube92_latlon_quart_90S90N/daily"
-urls(::Metadata{<:Any, <:ECCO4Monthly}) = "https://ecco.jpl.nasa.gov/drive/files/Version4/Release4/interp_monthly"
+metadata_url(m::Metadata{<:Any, <:ECCO2Daily})   = ECCO2_url *  "monthly/" * short_name(m) * "/" * metadata_filename(m)
+metadata_url(m::Metadata{<:Any, <:ECCO2Monthly}) = ECCO2_url *  "daily/"   * short_name(m) * "/" * metadata_filename(m)
+
+function metadata_url(m::Metadata{<:Any, <:ECCO4Monthly})
+    year = string(Dates.year(m.dates))
+    prefix =  "https://ecco.jpl.nasa.gov/drive/files/Version4/Release4/interp_monthly/"
```

Comment:
its nicer to use a top-level constant, which is easier to find for someone browing the source code, than to bury this in a function